scripts/validate_locale.ts

```diff
@@ -0,0 +1,128 @@
+const fs = require('fs');
+const path = require('path');
+
+// Read in parameters to choose which type of validation to run
+let args = process.argv.slice(2);
+let runType = args[0];
+if (!['checkLocaleUsage', 'compareLocales'].includes(runType)) {
+    console.log('No parameter or no valid run type. Please use checkLocaleUsage or compareLocales');
+}
+
+// Define all locales and primary locale
+const allLocales = ['cy-GB', 'en-GB', 'fr-FR', 'pl-PL', 'sv-SE'];
```

Comment:
This could also be programmatically determined by reading .json files from src/locales.